cpp/serve/function_table.cc

```diff
@@ -130,8 +130,10 @@ void FunctionTable::Init(String reload_lib_path, Device device, picojson::object
     this->_InitFunctions();
   } else {
     Module executable{nullptr};
-    if (false) {
-      // Todo(mlc-team): system lib reload // reload_lib_path starts with "system://"
+    if (StartsWith(reload_lib_path, "system://")) {
+      const PackedFunc* f_load_system_lib = Registry::Get("runtime.SystemLib");
+      ICHECK_NOTNULL(f_load_system_lib);
+      executable = (*f_load_system_lib)(std::string(reload_lib_path).substr(9));
```

Comment:
@tqchen Could you please help check if this implementation looks good? Seems we have no approach to test this at this moment.